slack-styles.user.css

```diff
@@ -34,3 +34,18 @@
     height: 32px;
     width: 32px;
 }
+
+/* Clearer "New Messages" divider */
```

Comment:
> Where do you get the color from btw?

It's the yellow part of the new Slack icon ;)

> Is it possible to steal the style from another part of the page just in case somebody has a theme set up?

That's probably a better idea. Although, making one element get its colors from another will either require JS or use of CSS variables.

Reply:
Either way this can be fixed later depending on how far this goes